app/controllers/posts_controller.rb

```diff
@@ -28,4 +28,18 @@ def create
       render :new
     end
   end
+

```

Comment:
- [ ] You should use `CanCanCan` for Post authorization in `Post Controller`, kindly make sure to implement this. you can add this line to your controller ``load_and_authorize_resource``

Reply:
Thank you @Ismailco for pointing out the issue. I have resolved it 😄